runtime/include/sandbox_set_as_initialized.h

```diff
@@ -34,7 +34,7 @@ sandbox_set_as_initialized(struct sandbox *sandbox, sandbox_state_t last_state)
 	}
 
 	/* State Change Bookkeeping */
-	assert(now > sandbox->timestamp_of.last_state_change);
+	assert(now >= sandbox->timestamp_of.last_state_change);
```

Comment:
This seems like it should be impossible. There is a fair amount of control flow between these state changes (essentially the body of `sandbox_init`. Any theories as to what might be going on here? Is `now` an actual timestamp? Makes me wonder if `__getcycles` isn't working.